blazor/pager/behaviour-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Behaviour Settings in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about basic functionalites of render the Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Behaviour Settings in Pager Component
+
+## Page size
+
+The Pager component provides an option to define the number of items to be displayed per page. This can be achieved by using the `PageSize` property. The default value of the `PageSize` property is "12".
+
+```csharp
+@using Syncfusion.Blazor.Navigations
+
+<SfPager PageSize="5">
+</SfPager>
+
+```
+
+## Page count
+
+The Pager component provides an option to define the number of numeric items to be displayed in the Pager for navigation. This can be achieved by using the `PageCount` property. The default value of the `PageCount` property is "10".
```

Comment:
The Pager component provides an option to define the number of numeric items to be displayed in the Pager for navigation. This can be achieved by using the `PageCount` property. The default value of the `PageCount` property is "10."